web_app/api/user.py

```diff
@@ -86,3 +88,27 @@ async def get_user_contract_address(wallet_id: str) -> GetUserContractAddressRes
         return {"contract_address": contract_address}
     else:
         return {"contract_address": None}
+
+
+@router.get("/api/get_stats", tags=["Statistics"], summary="Get all user amounts and the number of unique users.")
+async def get_stats():
+    """
+    Returns all user amounts for open positions and the number of unique users.
+
+    ### Returns:
+    - Dictionary containing:
+        - total_amounts: dict where keys are user IDs and values are total amounts
+        - unique_users: The count of unique users with open positions
+    """
+    try:
+        # Fetch total amounts for each user where position status is OPENED
+        total_amounts = position_db.get_all_amounts_for_opened_positions()
+        
+        # Get the number of unique users
+        unique_users_count = len(total_amounts)
```

Comment:
why do you do `len` for total_amounts, by variable name I understand that this is ready value, not amouns_objects. 
Please rename it and move `len` to `get_all_amounts_for_opened_positions` so then your variable name `total_amounts ` will be correct